roofit/roofitcore/inc/RooDataHist.h

```diff
@@ -250,7 +251,7 @@ class RooDataHist : public RooAbsData, public RooDirItem {
   mutable double* _sumw2{nullptr}; //[_arrSize] Sum of weights^2
   double*         _binv {nullptr}; //[_arrSize] Bin volume array
 
-  RooArgSet  _realVars ; // Real dimensions of the dataset 
+  RooArgSet  _realVars ; // Real dimensions of the dataset
```

Comment:
Try to keep formatting changes and code changes separate. (No need to change anything in the commit, though. 👍 )